app/models/APIResponse.scala

```diff
@@ -3,9 +3,19 @@ package models
 import io.circe._
 import io.circe.generic.semiauto._
 import io.circe.syntax._
+import models.db.ArticleWordCountResponse
 import play.api.Logger
 import play.api.mvc._
 
+
+case class WordCountAPIResponse(articlesWithoutCommissionedLength: List[ArticleWordCountResponse],
+                                articlesWithWordCountResponse: List[ArticleWordCountResponse],
```

Comment:
given `articlesWithoutCommissionedLength` and `articlesWithWordCountResponse` are of the same type I would argue that they should either both have `Response` on the end or both drop `Response`.